src/python/py/models/builder.py

```diff
@@ -1770,11 +1795,15 @@ def make_model(self, input_path):
                 self.make_layer(self.layer_id, module)
                 self.layer_id += 1
 
+            elif module.__class__.__name__.endswith("GLMBlock") and self.layer_id < self.num_layers:
```

Comment:
Can we merge this condition with the previous `elif` block and remove this case?

```python
elif (module.__class__.__name__.endswith("DecoderLayer") or module.__class__.__name__.endswith("GLMBlock")) and self.layer_id < self.num_layers:
```

Reply:
Great suggestion! Yes we can.